packages/client/src/modals/Transaction/Details.js

```diff
@@ -0,0 +1,114 @@
+        style={{ position: "relative", overflow: "hidden" }}
+      >
+        <Svg
+          name="LogoV"
+          style={{
+            position: "absolute",
+            top: "0px",
+            right: "-30px",
+            zIndex: 0,
+          }}
+        />
+        <div style={{ position: "relative", zIndex: 10 }}>
+          <label
+            className={`has-background-${statusColor} has-text-white px-3 py-2 rounded-lg mb-1 is-capitalized`}
+          >
+            {status}
+          </label>
+          <BannerInfo actionData={transaction} contacts={contacts} className="pl-4" />
+        </div>
+      </div>
+      <div className="px-5">
+        <div className="is-flex py-4 border-light-top">
+          <span className="has-text-grey flex-1">Executed On</span>
+          <div className="flex-1">
+            <b>
+              {new Date(date).toLocaleDateString("en-us")}{" "}
+              {new Date(date).toLocaleTimeString("en-us", { timeStyle: "short" })}
+            </b>
+          </div>
+        </div>
+        {(actionType === ACTION_TYPES.TRANSFER_TOKEN ||
+          actionType === ACTION_TYPES.TRANSFER_NFT) && (
+          <>
+            <div className="border-light-top is-flex py-4">
+              <div className="flex-1 has-text-grey">Sent from</div>
+              <div className="flex-1">
+                <div>
+                  <b>{safeData.name}</b>
+                </div>
+                <div>
+                  <span className="has-text-grey">{safeData.address}</span>
+                  <span className="pointer" onClick={() => clipboard.copy(safeData.address)}>
+                    <Svg name="Copy" className="mt-1 ml-2 pointer" />
+                  </span>
+                </div>
+              </div>
+            </div>
+            <div className="border-light-top is-flex py-4">
+              <div className="flex-1 has-text-grey">Sent to</div>
+              <div className="flex-1">
+                <div>
+                  <b>{recipient}</b>
+                  <span className="pointer" onClick={() => clipboard.copy(recipient)}>
+                    <Svg name="Copy" className="mt-1 ml-2 pointer" />
+                  </span>
+                </div>
+              </div>
+            </div>
+          </>
+        )}
+        {(actionType === ACTION_TYPES.ADD_SIGNER_UPDATE_THRESHOLD ||
+          actionType === ACTION_TYPES.REMOVE_SIGNER_UPDATE_THRESHOLD ||
+          actionType === ACTION_TYPES.UPDATE_THRESHOLD) && (
+          <div className="border-light-top is-flex py-4">
+            <div className="flex-1 has-text-grey">Signature Threshold</div>
+            <div className="flex-1 has-text-weight-bold">
+              {Object.keys(signers).length} of {safeData.threshold} owners
+            </div>
+          </div>
+        )}
+        <div className="border-light-top pb-4 has-text-grey">
+          <Signatures safeData={safeData} confirmations={signers} />

```

Comment:
Looks like Signatures only needs safeData.safeOwners, can we just pass safeOwners instead of the whole safeData?